resources/views/mship/feedback/view.blade.php

```diff
@@ -0,0 +1,21 @@
+@extends('layout')
+
+@section('content')
+<div class="panel panel-ukblue">
+	<div class="panel-heading"> My Feedback</div>
+	<div class="panel-body">
+		<p>Here, you can view feedback that has been sent to you by the VATSIM UK training departments.</p>
+        @foreach($feedback as $item)
```

Comment:
Personally, I would opt for a table layout here. I feel that it would be easier to read and browse through. I suggest something like this:
![image](https://user-images.githubusercontent.com/17804618/43026067-03c8d06c-8c6c-11e8-83bf-2be667a54e72.png)

Reply:
a24db50